### PR TITLE
Link to the repo

### PR DESCRIPTION
**Pull request by FlyingPhantom**
Added HTML tag to open the repository if an error appears

### CODE REVIEW

File: src/common/utils.js [hunk 1]
```diff
@@ -11,7 +11,7 @@ const renderError = (message, secondaryMessage = "") => {
     .gray { fill: #858585 }
     </style>
     <rect x="0.5" y="0.5" width="494" height="99%" rx="4.5" fill="#FFFEFE" stroke="#E4E2E2"/>
-    <text x="25" y="45" class="text">Something went wrong! file an issue at https://git.io/JJmN9</text>
+    <text x="25" y="45" class="text">Something went wrong! file an issue <a href="https://git.io/JJmN9" target="_blank">here</a></text>
```
Comment (anuraghazra) [inline src/common/utils.js:14]: Hi @FlyingPhantom Can we please make it

```html
<text x="25" y="45" class="text">Something went wrong! file an issue at <a href="https://git.io/JJmN9" target="_blank">https://git.io/JJmN9</a></text>
```

**Comment by FlyingPhantom** [comment]
Done @anuraghazra

**Review by AryanBeezadhur — APPROVED** [review]

**Comment by FlyingPhantom** [comment]
@anuraghazra  ??

**Comment by AryanBeezadhur** [comment]
@anuraghazra please merge this PR

**Pull request closed without merging**